workers/ohsome_quality_analyst/cli.py

```diff
@@ -165,9 +163,25 @@ def create_indicator(
                 force=force,
             )
         )
-        # TODO: Save as GeoJSON instead of printing to Stdout
+        if outfile:
+            if fid_field is None:
+                fid_field = DATASETS[dataset_name]["default"]
+            feature_collection = asyncio.run(
+                db_client.get_bpolys_from_db(dataset_name, feature_id, fid_field)
+            )
+            for feature in feature_collection.features:
+                if indicator.data is not None:
+                    feature["properties"].update(vars(indicator.data))
+                feature["properties"].update(vars(indicator.metadata))
+                feature["properties"].update(vars(indicator.result))
         click.echo(indicator.metadata)
         click.echo(indicator.result)
+    if outfile:
+        """Writing the output GeoJSON"""
```

Comment:
I think it is good to put this into a function in utils.

Reply:
will do it!